Code review video description

AntonioHincapie opened the pull request:
Construction and deployment of the capstone project 1
A. Set up the repository and tools:
- Added the family fonts.
- Created the JS files.
- Created the style file.
- Created the linters for HTML, CSS, and JS.
- Created the boilerplate HTML.
- Created the MIT license.
- Modify the README file.
- Created the gitignore.
B. Created the mobile version for the index and about:
- Added the body of the HTML file.
- Download the images for the background of the headline section and the program section.
- Downloaded the logo image.
- Write the content of the page.
- Downloaded the images for the program section.
- Added the style for the headline section.
- Added the style for the program section.
- Added style for the "featured-speakers" cards.
- Added style to the footer.
C. Setting up the desktop versions for the Main Page and the About page:
- Setting up the desktop version for the about page.
- Setting up the desktop version for the main page.
D. Added the mobile menu and deployed with js.
E. Deployed dynamic HTML and added the information of music bands:
- Added the data concerts.
- Added the images for the content.
- Added the dynamic HTML for these features.
F. Added live demo.
G. Added the link to the video description of the project.

AntonioHincapie committed:
Fixed console issue

-Added condition for apply the function.

AntonioHincapie committed:
Uploaded README and issue requirements

-Added the correct link for the COCOGOOSE font family.
-Added the link to the video description of the project.

Whoistolu commented on index.html:
```diff
@@ -6,7 +6,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Manizales Grita Rock (Capstone1)</title>
```
- [x] [REQUIRED] Great implementation for the pages, however, the height of the landing page seems too small. Kindly ensure that you pick the given height from the [behance template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) provided so that your landing page will look more like the template and more professional.
- Same applies to the landing page of the About section.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-04-21 16-25-41](https://user-images.githubusercontent.com/58692590/164497677-f01e2387-f426-4d82-9e2d-f0b39149abf2.png) | ![Screenshot from 2022-04-21 16-25-37](https://user-images.githubusercontent.com/58692590/164497730-4a75adc6-8e68-4381-b7e2-5ad4544635a4.png)

AntonioHincapie replied:
Hi @Whoistolu,
I see that the screen size that you used to see my implementation is so much wider than the Behance template. In the image above you can see that with a similar width it works the same.

![image](https://user-images.githubusercontent.com/99298594/164510266-5da4036f-5c0e-49de-bb5c-245df6b876ee.png)

Whoistolu commented on about.html:
```diff
@@ -6,7 +6,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>About us - Manizales Grita Rock (Capstone1)</title>
```
- [x] [REQUIRED] For the About section, you need to increase the height of the landing page so it makes your design look more professional and look more like that behance template provided.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-04-21 16-48-48](https://user-images.githubusercontent.com/58692590/164500375-c5938154-5ca7-4db1-8a9a-9129bae11dea.png) | ![Screenshot from 2022-04-21 16-48-59](https://user-images.githubusercontent.com/58692590/164500513-afbd876a-0ace-46ce-9c2c-b4b19cd255db.png)

AntonioHincapie replied:
Hi @Whoistolu,
Thanks for your comment, for the about page I have already increased the font size up to 50px.

![image](https://user-images.githubusercontent.com/99298594/164510786-ad402060-c18d-4b98-9ac9-5672cca4aedf.png)

Whoistolu replied:
Great implementation @AntonioHincapie 👍🏽

Whoistolu commented on about.html:
```diff
@@ -6,7 +6,11 @@
     <title>About us - Manizales Grita Rock (Capstone1)</title>
     <link rel="stylesheet" href="about_style.css" />
-    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link
```
- [x] [REQUIRED] In the about section, the logo section has a bit of excess height online the sample on the template, kindly reduce the height and ensure you stick closely to the behance template. Kindly ensure that right after the `h2` tag with the class='logo-title', there is a `p` tag just as it is on the behance because you have skipped that part.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-04-21 16-44-44](https://user-images.githubusercontent.com/58692590/164501236-84f9619b-523b-439d-bfc1-706bb4b65ae4.png) | ![Screenshot from 2022-04-21 16-44-48](https://user-images.githubusercontent.com/58692590/164501287-c50a1181-34eb-4cf1-9fd1-8d2042c1cb76.png)

Whoistolu reviewed (changes requested):
Hi @AntonioHincapie 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AntonioHincapie committed:
Fixed changes required.

-Added the text above the logo in the about page.
-Increased the font size for the headline in the about page.

AntonioHincapie commented:
Hi @chaw-bot,
Can you please clarify the required changes? I didn't understand how I should change the code.

AntonioHincapie commented:
@chaw-bot Yeah sure, I was asking for the changes related to the size of the home and about the landing page, you could do this by setting the section height to 100vh. I don't understand how to do it or what should I do here?.

AntonioHincapie committed:
Required changes

-Fixed contrast on the mobile menu.
-Fixed mobile menu for the about page.
-Added the 100vh height for the both pages.

lily-coder reviewed (approved):
Hi @AntonioHincapie, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

![image](https://user-images.githubusercontent.com/63964646/150861049-e057ddf0-f131-479b-8136-76ac0a3308f4.png)

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
